fetchers/kraken/index.ts

```diff
@@ -0,0 +1,61 @@
+import { Consola } from "consola";
+import axios from "axios";
+import { PriceDataFetched, Fetcher } from "../../types";
+
+const logger =
+  require("../../utils/logger")("fetchers/kraken") as Consola;
+const supportedPairs: string[] = require("./kraken-supported-pairs.json");
+
+const URL = "https://api.kraken.com/0/public/Ticker";
+
+const krakenFetcher: Fetcher = {
+  async fetchAll(tokenSymbols: string[]): Promise<PriceDataFetched[]> {
+    if (tokenSymbols.length === 0) {
```

Comment:
The same as before - should be moved to caller method to reduce code duplication